Packs/Core/Playbooks/playbook-Suspicious_Hidden_User_Created.yml

```diff
@@ -0,0 +1,1263 @@
+id: Suspicious Hidden User Created
+version: -1
+name: Suspicious Hidden User Created
+description: |-
+  This playbook addresses the following alerts:
+
+  - Suspicious Hidden User Created
+
+  Playbook Stages:
+
+  Triage:
+
+  - Retrieve event information about the created user
+
+  Investigation:
+
+  - Check if the user is local or domain
+  - For domain users: Retrieve AD attributes, including password expiration
+  - For local users: run a Powershell command to get "Password Expires" attribute of the local user
+  - Get risk level for the affected host
+  - Search for related Script Engine Activity alerts in the incident
+
+  Containment:
+
+  - Disable the suspicious user account (domain or local).
+  - If a related alert about malicious activity exists, kill the Causality Group Owner (CGO) process that created the suspicious user.
+
+  Requirements:
+
+  For response actions, you need the following integrations:
+
+  - Cortex Core - Investigation and Response
+  - Active Directory Query v2 (for domain user actions).
+tags:
+- T1136 - Create Account
+- 'T1564.002 - Hide Artifacts: Hidden Users'
```

Comment:
please remove quotes

Reply:
This is done by the platform when exporting the playbook, I assume because of the additional characters, so I'm inclined to keep it. On the tenant, the playbook's tags are normal